DuggaSys/courseed.js

```diff
@@ -237,23 +238,36 @@ function editVersion(cid, cname, ccode) {
 }
 
 function editSettings(){
```

Comment:
functions should have their opening brackets on a new line with same indentation as function declaration

Reply:
(does not apply to for-loops and if-statements, which all seems good in your code!)